### PR TITLE
[SPARK-20759] SCALA_VERSION in _config.yml should be consistent with  pom.xml

### PR DESCRIPTION
**Pull request by liu-zhaokun**
[https://issues.apache.org/jira/browse/SPARK-20759](https://issues.apache.org/jira/browse/SPARK-20759)
SCALA_VERSION in _config.yml is 2.11.7, but 2.11.8 in pom.xml. So I think SCALA_VERSION in _config.yml should be consistent with pom.xml.

### CODE REVIEW

**Comment by srowen** [comment]
@liu-zhaokun there are more changes to be made. Please make all the changes in https://github.com/apache/spark/pull/17982/files but don't update to 2.11.11; stay on 2.11.8.

**Comment by liu-zhaokun** [comment]
@srowen 
Hello, I have made some changes according to your advice.Please review the new PR?

File: repl/scala-2.11/src/main/scala/org/apache/spark/repl/SparkILoop.scala [hunk 1]
```diff
@@ -97,13 +97,13 @@ class SparkILoop(in0: Option[BufferedReader], out: JPrintWriter)
   override def commands: List[LoopCommand] = sparkStandardCommands
 
   /**
-   * We override `loadFiles` because we need to initialize Spark *before* the REPL
+   * We override `createInterpreter` because we need to initialize Spark *before* the REPL
```
Comment (srowen) [inline repl/scala-2.11/src/main/scala/org/apache/spark/repl/SparkILoop.scala]: Sorry, I meant you should follow the version changes. You can't take this change; it depends on 2.11.11

File: build/mvn [hunk 2]
```diff
@@ -91,13 +91,13 @@ install_mvn() {
 
 # Install zinc under the build/ folder
 install_zinc() {
-  local zinc_path="zinc-0.3.11/bin/zinc"
+  local zinc_path="zinc-0.3.15/bin/zinc"
```
Comment (srowen) [inline build/mvn]: Don't change the zinc version here

**Review by srowen — CHANGES_REQUESTED** [review]

**Comment by liu-zhaokun** [comment]
@srowen 
Thanks for your reply.I modified the files again.Please review it.

**Review by srowen — APPROVED** [review]

**Comment by liu-zhaokun** [comment]
@srowen 
Hi,could you help me merge this PR?

**Comment by srowen** [comment]
It isn't tested yet, no. Jenkins isn't working right now

**Comment by liu-zhaokun** [comment]
@srowen 
The test doesn't finish,need I do anything?

**Comment by liu-zhaokun** [comment]
@srowen  
Hello,do you know how to finish the test?

**Comment by liu-zhaokun** [comment]
@srowen 
Hi,do you know why this PR can't pass the test? I don't think it's my problem.

**Comment by srowen** [comment]
Merged to master/2.2/2.1

**Pull request closed without merging**